crates/runtime/benches/queries/README.md

```diff
@@ -406,5 +406,24 @@ select  cast(amc as FLOAT)/cast(pmc as FLOAT) am_pm_ratio
 
 | **Affected queries**     |                          |
 | ------------------------ | ------------------------ |
-| [q49.sql](tpcds/q49.sql) | [q75.sql](tpcds/q75.sql)|
-| [q90.sql](tpcds/q90.sql) |  |
+| [q49.sql](tpcds/q49.sql) | [q75.sql](tpcds/q75.sql) |
+| [q90.sql](tpcds/q90.sql) |                          |
+
+### Datafusion Physical Plan does not support logical expression `Exists`, `InSubquery` / not yet implemented for `GROUPING` aggregate function
+
+**Limitation**: Datafusion Physical Plan does not support `Exists`, `InSubquery`, `GROUPING` aggregate function, there will be errors when running the following queries in connector and in-memory arrow accelerator
```

Comment:
Which connector(s)?

Reply:
Is there any long-term or possible remedy?

Reply:
Updated.

Reply:
The long-term remedy would be having these aggregate functions implemented in Datafusion